OneCommunityGlobal/HighestGoodNetworkApp
# Shantanu unit test cases for colorsGenerator common component

# Description
Unit test case for color generator common component for pie chart

## How to test:
1. check into current branch
2. do npm install
3. npm test colorsGenerator.test.jsx
4. verify that all tests pass

## Screenshots or videos of changes:
![Shantanu-1](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/85691749/7fff5894-8421-498a-b971-50cf4f9ad480)
